__tests__/moment_spec.re

```diff
@@ -62,18 +62,9 @@ let () = {
         test
           "instantiation with format"
           (
-            fun () =>
-              Just (
-                Equal
-                  true
-                  (
-                    Moment.isSame
-                      (moment format::"YYYY-MM-DD" "2017-04-01")
-                      (moment format::"YYYY-MM-DD" "2017-04-01")
-                  )
-              )
+            fun () => Just (Equal true (Moment.isSame (moment "2017-04-01") (moment "2017-04-01")))
           );
-        test ".now" (fun () => Just (Equal true (moment_now () |> Moment.isValid)));
+        test ".now" (fun () => Just (Equal true (momentNow () |> Moment.isValid)));
```

Comment:
my intention was to put/translate creator / factory methods into top level functions and instance functions into module-scoped functions, does that make sense? @glennsl